Sources/OptionDescriptor.swift

```diff
@@ -869,6 +869,30 @@ struct _Descriptors {
         help: "Organize declarations by \"visibility\" (default) or \"type\"",
         keyPath: \.organizationMode
     )
+    let visibilityOrder = OptionDescriptor(
+        argumentName: "visibilityorder",
+        displayName: "Organization Order For Visibility",
+        help: "Order for visibility groups inside declaration",
+        keyPath: \.visibilityOrder
+    )
+    let typeOrder = OptionDescriptor(
+        argumentName: "typeorder",
+        displayName: "Organization Order For Declaration Types",
+        help: "Order for declaration type groups inside declaration",
+        keyPath: \.typeOrder
+    )
+    let customVisibilityMarks = OptionDescriptor(
+        argumentName: "visibilitymarks",
+        displayName: "Custom Visibility Marks",
+        help: "Mark for concrete visibility group (public_Public Fields)",
```

Comment:
I've used the `:` syntax as a separator in this sort of mapping argument before. I think that would be a bit better than an underscore. Also, including `,...` could be helpful here to indicate that it supports multiple values.

```suggestion
        help: "Mark for concrete visibility group (public:Public Fields,...)",
```